### PR TITLE
[TEC-3332] Ensure that venue state or province are displayed next to the city in list view.

### PR DESCRIPTION
**Pull request by EugeneKyale**
Ticket : https://theeventscalendar.atlassian.net/browse/TEC-3332
Artifact : https://www.loom.com/share/4bb7d07f03954fa8bfd9b2458ef54351

### CODE REVIEW

File: src/views/v2/list/event/venue.php [hunk 1]
```diff
@@ -31,8 +31,13 @@
 	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
 		<?php echo esc_html( $address ); ?>
+
+		<?php if ( $venue->city ) : ?>
+			<?php echo esc_html( $venue->city . $separator ) ?>
+		<?php endif; ?>
+		
 		<?php if ( $append_after_address ) : ?>
```
Comment (paulmskim) [inline src/views/v2/list/event/venue.php:39]: a lot of open and close php tags, let's make this under one open and close tag

File: src/views/v2/list/event/venue.php [hunk 2]
```diff
@@ -31,8 +31,13 @@
 	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
 		<?php echo esc_html( $address ); ?>
+
+		<?php if ( $venue->city ) : ?>
```
Comment (paulmskim) [inline src/views/v2/list/event/venue.php:35]: check if empty rather than just checking venue city

File: src/views/v2/list/event/venue.php [hunk 3]
```diff
@@ -31,8 +31,13 @@
...
+			<?php echo esc_html( $venue->city . $separator ) ?>
+		<?php endif; ?>
+		
 		<?php if ( $append_after_address ) : ?>
-			<?php echo esc_html( reset( $append_after_address ) ); ?>
+			<?php echo esc_html( reset( $append_after_address ) ) ?>
```
Comment (paulmskim) [inline src/views/v2/list/event/venue.php:40]: keep semicolon

**Review by paulmskim — CHANGES_REQUESTED** [review]

File: src/views/v2/list/event/venue.php (hunk 1 above)
Comment (EugeneKyale) [reply src/views/v2/list/event/venue.php]: I've refactored this to be under one open and close tag

File: src/views/v2/list/event/venue.php (hunk 2 above)
Comment (EugeneKyale) [reply src/views/v2/list/event/venue.php]: I've updated this to use if ! empty

File: src/views/v2/list/event/venue.php (hunk 3 above)
Comment (EugeneKyale) [reply src/views/v2/list/event/venue.php]: should be updated

File: src/views/v2/list/event/venue.php [hunk 4]
```diff
@@ -22,17 +22,24 @@
...
+			echo esc_html( $address ); 
+
+			if ( ! empty( $venue->city ) ) : 
+				echo esc_html( $venue->city . $separator ); 
+		 	endif;
+		
+		 	if ( $append_after_address ) : 
+				echo esc_html( reset( $append_after_address ) ); 
+		 	endif; 
+		?>
```
Comment (paulmskim) [inline src/views/v2/list/event/venue.php:43]:
```suggestion
		<?php 
		echo esc_html( $address ); 

		if ( ! empty( $venue->city ) ) : 
			echo esc_html( $venue->city . $separator ); 
		endif;

		if ( $append_after_address ) : 
			echo esc_html( reset( $append_after_address ) ); 
		endif; 
		?>
```

**Review by paulmskim — APPROVED** [review]

**Pull request merged**